auto-test-engine/src/main/java/com/auto/solution/TestEngine/TestEngineHelper.java

```diff
@@ -13,7 +13,7 @@ void updateTargetAndResourcesIfProvided(ResourceManager rm){
 			String resourceDirectoryProvided = System.getProperty("resourcedirectory");
 			String targetProvided = System.getProperty("targetname");
 			if(resourceDirectoryProvided != null){
-				rm.setResourcesBaseLocationRelativeToProjectBase(resourceDirectoryProvided);
+				rm.setResourcesBaseLocation(resourceDirectoryProvided);
```

Comment:
Couple of questions around this:
1. So now, the 'resourcename' must always be the path, instead of the name of resource folder?
2. if yes, then we need to support the name as well. As in most of the cases we will require resource folder name instead of the absolute path.